### ISSUE TITLE
liquid transport, speed freez at 18km/h after unlaoding

### ISSUE THREAD

**Issue by andreasthu** [Courseplay/courseplay#2532]
Hello guys.
first, happy new year

but now i have a problem (v 02.0050)
i use the MAN TGS ITRunner and the hooklift-container from the Kotte Universal Pack 1.0.0.8 to transport and sell digestate and gülle from cow, pigs and bga
http://www.farming-simulator.com/mod.php?lang=de&country=de&mod_id=71421&title=fs2017
http://forbidden-mods.phpbb8.de/modpacks-f1875/kotte-universal-pack-t33442.html
my problem is, that the speed from the courseplay driver freezes to 18 km/h after unlaoding/selling the gülle or digestate at the sell station.
the speed freezes to 18km/h and he drives not faster.
only fire the cp-helper and new hire him the speed goes up faster - but only to the next seling action, then the speed is also max 18 km/h.
can the speed setting "Wendemanöver" the cause?

![fsscreen_2018_01_01_13_38_30](https://user-images.githubusercontent.com/24860501/34467857-a6a836f0-eefc-11e7-88ae-a7844446a960.png)

here you can see, he must drive faster:
```
    <waypoint80 angle="-171.88" speed="31" pos="-137.65 -89.29"/>
    <waypoint81 angle="-175.18" speed="36" pos="-139.06 -99.21"/>
    <waypoint82 angle="179.89" speed="34" pos="-139.90 -109.22"/>
    <waypoint83 angle="174.82" speed="32" pos="-139.89 -114.67"/>
    <waypoint84 angle="171.11" speed="27" pos="-139.22 -122.15"/>
    <waypoint85 angle="171.46" speed="12" pos="-137.67 -132.04"/>
    <waypoint86 angle="-6.39" speed="0" pos="-137.51 -133.12"/>
    <waypoint87 angle="0.69" speed="16" rev="1" pos="-137.88 -130.76"/>
    <waypoint88 angle="1.08" speed="15" rev="1" pos="-138.25 -128.73"/>
    <waypoint89 angle="-19.68" speed="17" rev="1" pos="-138.71 -126.67"/>
    <waypoint90 angle="-24.19" speed="17" rev="1" pos="-139.39 -124.78"/>
    <waypoint91 angle="-31.24" speed="17" rev="1" pos="-140.23 -122.90"/>
    <waypoint92 angle="-58.05" speed="16" rev="1" pos="-141.28 -121.16"/>
    <waypoint93 angle="-63.95" speed="18" rev="1" pos="-143.04 -120.07"/>
    <waypoint94 angle="-62.74" speed="16" rev="1" pos="-144.86 -119.18"/>
    <waypoint95 angle="-73.22" speed="17" rev="1" pos="-146.65 -118.26"/>
    <waypoint96 angle="-76.11" speed="21" rev="1" pos="-148.62 -117.66"/>
    <waypoint97 angle="-90.01" speed="17" rev="1" pos="-150.56 -117.18"/>
    <waypoint98 angle="-88.51" speed="11" rev="1" pos="-154.58 -117.18"/>
    <waypoint99 angle="-95.98" speed="12" rev="1" pos="-156.61 -117.13"/>
    <waypoint100 angle="-96.09" speed="1" rev="1" pos="-165.55 -117.88"/>
    <waypoint101 angle="83.94" speed="0" rev="1" pos="-169.54 -118.25" wait="1"/>
    <waypoint102 angle="78.58" speed="15" pos="-156.23 -117.18"/>
    <waypoint103 angle="64.58" speed="19" pos="-150.37 -116.00"/>
    <waypoint104 angle="64.01" speed="22" pos="-148.98 -115.34"/>
    <waypoint105 angle="47.73" speed="24" pos="-146.93 -114.34"/>
    <waypoint106 angle="30.15" speed="26" pos="-145.76 -113.28"/>
    <waypoint107 angle="17.43" speed="28" pos="-144.99 -111.96"/>
    <waypoint108 angle="15.54" speed="23" pos="-144.51 -110.43"/>
    <waypoint109 angle="4.27" speed="25" pos="-144.11 -108.97"/>
    <waypoint110 angle="5.02" speed="25" pos="-143.99 -107.41"/>
    <waypoint111 angle="12.76" speed="23" pos="-143.85 -105.83"/>
    <waypoint112 angle="6.96" speed="36" pos="-141.64 -96.06"/>
    <waypoint113 angle="3.09" speed="42" pos="-140.60 -87.59"/>
    <waypoint114 angle="1.28" speed="45" pos="-140.06 -77.39"/>
    <waypoint115 angle="0.26" speed="51" pos="-139.83 -67.25"/>
    <waypoint116 angle="0.46" speed="55" pos="-139.78 -56.89"/>
```

here is the complete course:
[courseStorage0063.zip](https://github.com/Courseplay/courseplay/files/1596039/courseStorage0063.zip)

lg
andreas

**Comment by pops64** [comment]
No. It probably still thinks it should be unloading. That is the speed setting we use when we detect a place to unload. We will need a log with a debug channel 21 and another give me about hour I will give you better instructions.

**Comment by andreasthu** [comment]
ok thanks,
here is a zip with the log with debugmod 21+23
and 4pics, 1 during unload and 3 after
[Desktop.zip](https://github.com/Courseplay/courseplay/files/1596504/Desktop.zip)

lg
andreas

**Comment by andreasthu** [comment]
ok, thanks
will this bug be fixed in future or is this so as it is?
while then i sell all my hooklift containesr and must drive in all the coures new with an other trailer 
lg
andreas

**Comment by pops64** [comment]
Closed #2909
